e2e/ts_library/reference_types_directive/tsconfig_types.ts

```diff
@@ -1,3 +1,4 @@
+describe('@jasmine/types types are included via tsconfig.json "types"=["jasmine"]', () => {});
```

Comment:
should this be empty? just a stray addition?

Reply:
Just testing that `describe` is a valid symbol. I'll clean it up.